api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -9,34 +9,105 @@
  */
 package org.openmrs.notification.db.hibernate;
 
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+
+import org.hibernate.SessionFactory;
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.openmrs.api.context.Context;
+import org.openmrs.notification.Alert;
 import org.openmrs.notification.AlertService;
+import org.openmrs.notification.db.AlertDAO;
 import org.openmrs.test.BaseContextSensitiveTest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * This class tests the hibernate alert data access. TODO Consider changing this and all subsequent
- * tests to use dbunit
+ * This class tests the hibernate alert data access. TODO Consider changing this
+ * and all subsequent tests to use dbunit
  */
 public class HibernateAlertDAOTest extends BaseContextSensitiveTest {
-	
+
+	private static final Logger log = LoggerFactory.getLogger(HibernateAlertDAOTest.class);
+	private HibernateAlertDAO hibernateAlertDAO;
+	private Object getAllAlerts;
+	private Object alertId;
+
 	@Before
 	public void runBeforeEachTest() {
+
+		if (hibernateAlertDAO == null)
+			hibernateAlertDAO = (HibernateAlertDAO) applicationContext.getBean("HibernateAlertDao");
 		authenticate();
 	}
-	
+
 	/**
 	 * Test that you can get alerts
 	 * 
 	 * @throws Exception
 	 */
+	// @Test
+	// public void shouldGetAlerts() {
+
+	// }
+    @Test
+	public void getAllAlerts_shouldReturnTrueIfRequested() {
+		AlertService alertService = Context.getAlertService();
+		if (getAllAlerts != null) {
+			Assert.assertEquals(" ", alertService.getAllAlerts(), alertService.getClass());
+		}
+	}
+    @Test
+	public void getAlert_shouldGetAlertById(){
+		assertEquals(1, AlertService.getAlertId();
+	}
 	@Test
-	public void shouldGetAlerts() {
-		
-		AlertService as = Context.getAlertService();
-		//System.out.println(as.getAllAlerts());
-		
+	public void saveAlert_shouldReturnSavedAlertObjectInDatabase() {
+		Session session = null;
+		Transaction transaction =null;
+		  try {
+		      session = HibernateUtil.getSessionFactory().openSession();
+		      transaction = session.beginTransaction();
+		      transaction.begin();
+		      Alert alert = new Alert();
+		      alert.setAlertsByRole("Alert role meesage",alertRole.getMessage());
```

Comment:
Please provide a space after ','

Reply:
Ok sure @varung thanks, i wanted to squash commits but it seems i cant squash commits because my first PR is not recognised by git